nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs

```diff
@@ -1441,6 +1466,32 @@ private void ResponseUnlockWorld(ActionBase.ActionEvaluation<UnlockWorld> eval)
             }
         }
 
+        private void ResponseHackAndSlashRandomBuff(ActionBase.ActionEvaluation<HackAndSlashRandomBuff> eval)
+        {
+            if (!(eval.Exception is null))
+            {
+                Debug.LogError($"HackAndSlashRandomBuff exc : {eval.Exception.InnerException}");
+                return;
+            }
+
+            UpdateCurrentAvatarStateAsync(eval).Forget();
+            UpdateAgentStateAsync(eval).Forget();
+            UpdateHackAndSlashBuffState(eval);
```

Comment:
If the UpdateHackAndSlashBuffState() function will be async function,

```suggestion
        private UniTaskVoid ResponseHackAndSlashRandomBuffAsync(ActionBase.ActionEvaluation<HackAndSlashRandomBuff> eval)
        {
            if (!(eval.Exception is null))
            {
                Debug.LogError($"HackAndSlashRandomBuff exc : {eval.Exception.InnerException}");
                return;
            }

            UniTask.WhenAll(
                UpdateCurrentAvatarStateAsync(eval),
                UpdateAgentStateAsync(eval),
                UpdateHackAndSlashBuffStateAsync(eval)
            );
```

or just forget that.

```
UpdateHackAndSlashBuffStateAsync(eval).Forget();
```